File: src/main/scala/de/mineformers/allomancy/internal/network/ToggleBurningMetal.scala
package de.mineformers.allomancy.internal.network

import tessera.io.Message

/**
  * ToggleBurningMetal
  *
  * @author PaleoCrafter
  */
case class ToggleBurningMetal(val metal: String) extends Message {
  def this() = this(null)
}

File: src/main/java/de/mineformers/allomancy/network/messages/ToggleBurningMetal.java
package de.mineformers.allomancy.network.messages;

import de.mineformers.allomancy.network.Message;

/**
 * ToggleBurningMetal
 *
 * @author PaleoCrafter
 */
public class ToggleBurningMetal extends Message
{
    public String metal;

    public ToggleBurningMetal()
    {
    }

    public ToggleBurningMetal(String metal)
    {
        this.metal = metal;
    }
}
